### DISCUSSION TITLE
Adding Style to simple-datatables-components

### DISCUSSION THREAD

**Discussion by kevxard** [vincjo/svelte-simple-datatables#52] [Q&A] [answered]
Hello! Greetings from Colombia.

I'm trying to add styles to datatable, styles like a divider between td's, change text pagination row count color, or delete the space between the last element of the page and the page end, but I can't.

Could you help me with that? :)  I would be very grateful if you do!

PD: This is my style progress 
![image](https://user-images.githubusercontent.com/48103810/167903060-bf86ebbf-50ef-43d8-a568-b45cc6de72d5.png)

and this is my expected result
![image](https://user-images.githubusercontent.com/48103810/167903694-d399c7dc-314e-4593-9893-ff0fa0708f1c.png)

**Comment by vincjo** [comment, marked as answer]
Hello Kyovint,

You've probably already done it but the simplest I think is to desactivate the style.
````js
const settings = { css: false } 
````
Below is a dark template on which you can rely to adapt the style to your needs
https://github.com/vincjo/svelte-simple-datatables/blob/master/example/public/datatable.dark.css

You can find example [here](https://vincjo.fr/svelte-simple-datatables/#/demo/css)

**Reply by kevxard** [reply]
You're the best! thanks!

**Comment by louislugas** [comment]
Hi @vincjo , could you guide how to import/rewrite the custom css? I read from the custom/dark css from the example, but it can't find the .datatables class
